homeassistant/components/camera/__init__.py

```diff
@@ -614,6 +672,34 @@ def async_update_token(self) -> None:
             hashlib.sha256(_RND.getrandbits(256).to_bytes(32, "little")).hexdigest()
         )
 
+    async def async_internal_added_to_hass(self) -> None:
+        """Run when entity about to be added to hass."""
+        await super().async_internal_added_to_hass()
```

Comment:
Isn't this a pointless method? Is there a reason you add it?

Reply:
Thanks for catching. Whoops, when refactoring I dropped an important call to refresh the webrtc providers. This is for the case where the camera is added after the provider is already registered.

I attempted to write a test to catch this, but it is difficult given the way the test setup happens registering a camera at the same time as `camera` is initialized only. I can consider test coverage for this when adding on real usage.